CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 * Content-Length response bodies with lengths greater than Integer.MAX_VALUE report null length
   * Previously the OkhttpClient would throw an exception, and ApacheHttpClient
     would report a wrong, possibly negative value
+* The `headers` map in the `Response` object now lower-cases header names. It is now a `TreeMap` with
```

Comment:
maybe something like, which helps with motivation and talks about impact vs impl.

```
* `Response.headers` is now case-insensitive with regards to keys, and iterates in lexicographic order.
  * This better supports http2, which down-cases header names.
```